cmd/neofs-lens/internal/objects/link.go

```diff
@@ -0,0 +1,49 @@
+package objects
+
+import (
+	"errors"
+	"os"
+
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
+	"github.com/spf13/cobra"
+)
+
+var linkCMD = &cobra.Command{
+	Use:   "link",
+	Short: "Inspect link object",
+	Run:   linkFunc,
+}
+
+func linkFunc(cmd *cobra.Command, _ []string) {
+	if vPath == "" {
+		common.ExitOnErr(cmd, errors.New("empty path to file"))
+	}
+
+	raw, err := os.ReadFile(vPath)
+	common.ExitOnErr(cmd, common.Errf("reading file: %w", err))
+
+	var link object.Link
+	err = link.Unmarshal(raw)
+	if err != nil {
+		cmd.Printf("%q file does not contain raw link payload, trying to decode it as a full NeoFS object", vPath)
+
+		var obj object.Object
+		err = obj.Unmarshal(raw)
+		common.ExitOnErr(cmd, common.Errf("decoding NeoFS object: %w", err))
+
+		raw = obj.Payload()
+		err = link.Unmarshal(raw)
```

Comment:
cool! this can also be used for contents of storages like FS tree

btw
```suggestion
		err = obj.ReadLink(&link)
```
'd be even more laconic, huh?

and since the method `Makes sense only if object has [TypeLock] type`, it's also worth to either pre-check object type or include type to error context on decode failure